File: capstone-project/capstone-project-client/src/Ingredient/Ingredient.js

function Ingredient(ingredientData ) {

    return (
        //todo: include the unit of measurement for quantity    
        <>
        <div>{ingredientData.ingredientData.food}</div>
        {/* : {(ingredientData.ingredientData.quantity).toFixed(2)} {ingredientData.ingredientData.measure} */}
        {/* <div>{ingredientData.ingredientData.foodId}</div>
        <div>{ingredientData.ingredientData.foodCategory}</div>
        <div>{ingredientData.ingredientData.quantity}</div>
        <div>{ingredientData.ingredientData.measure}</div> */}
        </>
    );
}

export default Ingredient;

File: capstone-project/capstone-project-server/src/main/java/learn/recipemanager/models/Ingredient.java
package learn.recipemanager.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Ingredient {

    private String foodId;
    private String name;
    private String foodCategory;
    private double quantity;
    private String measure;
}
